tests/integration/bank/keeper/keeper_test.go

```diff
@@ -983,160 +1050,172 @@ func (suite *IntegrationTestSuite) TestUndelegateCoins() {
+	assert.Equal(t, metadata[1].GetDescription(), actualMetadata.GetDescription())
+	assert.Equal(t, metadata[1].GetDenomUnits()[1].GetDenom(), actualMetadata.GetDenomUnits()[1].GetDenom())
+	assert.Equal(t, metadata[1].GetDenomUnits()[1].GetExponent(), actualMetadata.GetDenomUnits()[1].GetExponent())
+	assert.DeepEqual(t, metadata[1].GetDenomUnits()[1].GetAliases(), actualMetadata.GetDenomUnits()[1].GetAliases())
 }
 
-func (suite *IntegrationTestSuite) TestIterateAllDenomMetaData() {
-	ctx := suite.ctx
+func TestIterateAllDenomMetaData(t *testing.T) {
+	f := initFixture(t)
+	t.Parallel()
 
-	expectedMetadata := suite.getTestMetadata()
+	ctx := f.ctx
+
+	expectedMetadata := getTestMetadata()
 	// set metadata
 	for i := range []int{1, 2} {
-		suite.bankKeeper.SetDenomMetaData(ctx, expectedMetadata[i])
+		f.bankKeeper.SetDenomMetaData(ctx, expectedMetadata[i])
 	}
 	// retrieve metadata
 	actualMetadata := make([]types.Metadata, 0)
-	suite.bankKeeper.IterateAllDenomMetaData(ctx, func(metadata types.Metadata) bool {
+	f.bankKeeper.IterateAllDenomMetaData(ctx, func(metadata types.Metadata) bool {
 		actualMetadata = append(actualMetadata, metadata)
 		return false
 	})
 	// execute checks
 	for i := range []int{1, 2} {
-		suite.Require().Equal(expectedMetadata[i].GetBase(), actualMetadata[i].GetBase())
-		suite.Require().Equal(expectedMetadata[i].GetDisplay(), actualMetadata[i].GetDisplay())
-		suite.Require().Equal(expectedMetadata[i].GetDescription(), actualMetadata[i].GetDescription())
-		suite.Require().Equal(expectedMetadata[i].GetDenomUnits()[1].GetDenom(), actualMetadata[i].GetDenomUnits()[1].GetDenom())
-		suite.Require().Equal(expectedMetadata[i].GetDenomUnits()[1].GetExponent(), actualMetadata[i].GetDenomUnits()[1].GetExponent())
-		suite.Require().Equal(expectedMetadata[i].GetDenomUnits()[1].GetAliases(), actualMetadata[i].GetDenomUnits()[1].GetAliases())
+		assert.Equal(t, expectedMetadata[i].GetBase(), actualMetadata[i].GetBase())
+		assert.Equal(t, expectedMetadata[i].GetDisplay(), actualMetadata[i].GetDisplay())
+		assert.Equal(t, expectedMetadata[i].GetDescription(), actualMetadata[i].GetDescription())
+		assert.Equal(t, expectedMetadata[i].GetDenomUnits()[1].GetDenom(), actualMetadata[i].GetDenomUnits()[1].GetDenom())
+		assert.Equal(t, expectedMetadata[i].GetDenomUnits()[1].GetExponent(), actualMetadata[i].GetDenomUnits()[1].GetExponent())
+		assert.DeepEqual(t, expectedMetadata[i].GetDenomUnits()[1].GetAliases(), actualMetadata[i].GetDenomUnits()[1].GetAliases())
 	}
 }
 
-func (suite *IntegrationTestSuite) TestBalanceTrackingEvents() {
+func TestBalanceTrackingEvents(t *testing.T) {
+	f := initFixture(t)
+	t.Parallel()
+
 	// replace account keeper and bank keeper otherwise the account keeper won't be aware of the
 	// existence of the new module account because GetModuleAccount checks for the existence via
 	// permissions map and not via state... weird
 	maccPerms := simapp.GetMaccPerms()
```

Comment:
Is there a way we can remove the simapp import? Don't we need only 3 accounts here given we use 3 modules?